.pre-commit-config.yaml

```diff
@@ -43,3 +46,16 @@ repos:
       rev: 0.9.1
       hooks:
           - id: pyproject-fmt
+
+    # Backstage linter
+    - repo: local
```

Comment:
Is there a native hook avaliable?

Reply:
nope, unfortunately, the best thing we have now.

Reply:
Would be good to run `pre-commit` in the checks, it found a couple of problems in the current code base